MaxText/configs/base.yml

```diff
@@ -654,14 +654,13 @@ sa_v_layout: "HEAD_DIM_MINOR"
 ### Determine if we want to use load balance for context parallelism
 context_parallel_load_balance: True
 
-#######################
 ### Paged Attention ###
-#######################
 # These settings take effect only when `attention=paged`.
 # They should be adjusted based on the available HBM and model config.
 pagedattn_num_pages: 64
 pagedattn_tokens_per_page: 32
-pagedattn_pages_per_compute_block: 8
+pagedattn_pages_per_compute_block: 4
+pagedattn_max_pages_per_group: -1  # defaults to (max_target_length + pa_tokens_per_page - 1) // pa_tokens_per_page
```

Comment:
nit: An explicit comment saying default is one group for all page, this saves reader from parsing and understanding the equation. Fine to keep as is.

Reply:
Yes.

Correct. I meant one page group per max target seq